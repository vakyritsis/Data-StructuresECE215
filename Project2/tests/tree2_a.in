93 66 90 16 23 33 36 57 71 22 69 79 56 9 46 4 8 40 17 28 84 31 89 12 30 76 10 94 1 19 25 48 45 35 54 74 92 15 68 43 75 58 7 6 86 29 32 18 21 37 99 65 47 63 2 70 52 97 72 78 85 60 5 95 -1
