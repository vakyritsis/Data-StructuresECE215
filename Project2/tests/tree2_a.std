47 22 10 6 4 2 1 5 8 7 9 17 15 12 16 19 18 21 33 29 25 23 28 31 30 32 40 36 35 37 45 43 46 74 63 56 52 48 54 58 57 60 69 66 65 68 71 70 72 89 79 76 75 78 85 84 86 94 92 90 93 97 95 99 
