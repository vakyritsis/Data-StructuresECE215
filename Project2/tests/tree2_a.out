46 21 9 5 2 1 4 7 6 8 16 12 10 15 18 17 19 32 28 23 22 25 30 29 31 37 35 33 36 43 40 45 72 60 54 48 47 52 57 56 58 68 65 63 66 70 69 71 86 78 75 74 76 84 79 85 93 90 89 92 95 94 97 99 